doc/changelog.rst

```diff
@@ -1,6 +1,27 @@
 Changelog
 =========
 
+2.0.0a0
+-------
+
+Breaking changes:
+* Added support for Python 3.12
```

Comment:
* Update for compatibility with Flask 3.
* Update for compatibility with WTForms 3.

Reply:
These aren't breaking changes, and are already listed further down 👍